PowerShellGet/PSModule.psm1

```
@@ -10856,14 +10963,66 @@ function Install-PackageUtility
+                        {
+                            $message = $LocalizedData.ForceAcceptLicense -f $pkg.Name
+
+                            ThrowError -ExceptionName "System.ArgumentException" `
+                                       -ExceptionMessage $message `
+                                       -ErrorId "ForceAcceptLicense" `
+                                       -CallerPSCmdlet $PSCmdlet `
+                                       -ErrorCategory InvalidArgument
+                        }
+
+                        If (-not ($YesToAll -or $NoToAll -or $AcceptLicense))
+                        {
+                            if(-not(Test-Path -path "$sourceModulePath\License.txt" -PathType Leaf))
+                            {
+                                $message = $LocalizedData.LicenseTxtNotFound
+
+                                ThrowError -ExceptionName "System.ArgumentException" `
+                                           -ExceptionMessage $message `
+                                           -ErrorId "LicenseTxtNotFound" `
+                                           -CallerPSCmdlet $PSCmdlet `
+                                           -ErrorCategory InvalidArgument
+                            }
+                            $EULA = Get-Content "$sourceModulePath\License.txt"
+                            $FormattedEula = ""
+                            foreach($line in $EULA)
+                            {
+                                $FormattedEula +=$line -join "`r`n"
+                                $FormattedEula +="`r`n"
+                            }
+                            $message = $LocalizedData.AcceptanceLicenseQuery -f $pkg.Name
+                            $result = $request.ShouldContinue($message, $FormattedEula, [ref]$yesToAll, [ref]$NoToAll)
+                            if(($result -eq $false) -or ($NoToAll -eq $true))
+                            {
+                                return

```

>return [](start = 32, length = 6)

We should throw an error and return when user declined the prompt.